iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/HvdcLineImpl.java

```diff
@@ -231,6 +241,145 @@ public void remove() {
         network.getListeners().notifyAfterRemoval(id);
     }
 
+    @Override
+    public boolean connect() {
+        return connect(SwitchPredicates.IS_NONFICTIONAL_BREAKER);
```

Comment:
didn't realize, but isn't it fictitious rather than fictional?

Reply:
both can be used, IMO.
If we want to change it, we should do it in another PR